dotCMS/src/main/java/com/dotcms/contenttype/model/field/Field.java

```diff
@@ -302,6 +302,18 @@ public Optional<FieldValueBuilder> fieldValue(final Object value) {
     return Optional.empty();
   }
 
+  public Optional<String> fieldVariableValue(final String fieldVariableName) {
+    final List<FieldVariable> fieldVariables = this.fieldVariables();
+
+    for (final FieldVariable fieldVariable : fieldVariables) {
+      if (fieldVariable.key().equals(fieldVariableName)) {
+        return Optional.of(fieldVariable.value());
+      }
+    }
+
+    return Optional.empty();
+  }
+
   ;

```

Comment:
What about this lonely semicolon?

Reply:
done https://github.com/dotCMS/core/pull/22304/commits/fc2a9966e4ffd8e1e5a0ebb351a0d02e13d4be9f